File: temperature/weatherServer/model/weather.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var WeatherSchema = new Schema({
    sensorType: {
        type: String
    },
    temperature: {
        type: Number,
        required: true
    },
    humidity: {
        type: Number,
        required: true
    },
    created_at: {
        type: Date,
        default: Date.now
    }
});

var Weather = module.exports = mongoose.model("weather", WeatherSchema);

File: temperature/TempHum/app/src/main/java/com/testmyown/temphum/data/model/Weather.java
package com.testmyown.temphum.data.model;

/**
 * Created by ledinhduy on 17/09/2017.
 */

public class Weather {
    public double temperature;
    public double humidity;

    public Weather(double temperature, double humidity) {
        this.temperature = temperature;
        this.humidity = humidity;
    }

    public Weather() {

    }
}
